workers/unity/Packages/com.improbable.gdk.core/Worker/WorkerConnector.cs

```diff
@@ -186,6 +187,16 @@ await Worker.CreateWorkerAsync(GetAlphaLocatorConfig(), connectionParameters, lo
         /// <returns>A <see cref="ReceptionistConfig"/> object.</returns>
         protected abstract ReceptionistConfig GetReceptionistConfig(string workerType);
 
+        protected virtual string GetPlayerId()
+        {
+            return null;
+        }
+
+        protected virtual string GetDisplayName()
+        {
+            return null;
+        }
```

Comment:
I know this similar to other functions defined here - but I feel we should either provide a reasonable default implementation or make these fully `abstract`.

Returning null as a default implementation seems likely to break things later on.